### PR TITLE
DT-90 <feat: history page ui>

### PR DESCRIPTION
**Pull request by Duy-Tinh-Dev**
DT-90 <feat: history page ui>

### CODE REVIEW

**Comment by Duy-Tinh-Dev** [comment]
@Giang-st1-00 @lamxuanthinh help me check

File: components/common/History/Payload/index.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import Image from "next/image";
+import Button from "../../Button";
```
Comment (lamxuanthinh) [inline components/common/History/Payload/index.tsx]: why you don't use `@/` @DuyTinh2003

**Comment by lamxuanthinh** [comment]
Please write description more detail @DuyTinh2003

**Comment by Giang-st1-00** [comment]
ok

**Comment by lamxuanthinh** [comment]
oke

**Pull request merged**